.github/workflows/taqueria-docker-publish.yml

```diff
@@ -0,0 +1,115 @@
+name: Taqueria Docker image
+on:
+  push:
+    branches:
+      - 'main'
+    tags:
+      - 'v*'
+  pull_request:
+    branches:
+      - 'main'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.ref != 'refs/heads/main' || github.run_number }}
+  cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
+
+env:
+  REGISTRY: ghcr.io
+
+jobs:
+
+  binary-for-image:
+    name: Build taq binary
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    if: ${{ github.event_name != 'release' && !github.event.pull_request.head.repo.fork }}
+    env:
+      DENO_TARGET: x86_64-unknown-linux-gnu
+
+    steps:
+      - name: Setup repo
+        uses: actions/checkout@v2
+
+      - name: Setup Deno
+        uses: denoland/setup-deno@v1
+        with:
+          deno-version: "1.23.4"
+
+      - uses: actions/cache@v3
+        with:
+          path: |
+            ./deno
+            ~/.cache/deno
+          key: ${{ runner.os }}-deno-${{ hashFiles('**/deno-lock.json') }}
+
+      - name: Build the binary
+        id: build-bin
+        shell: bash
+        env:
+          TAQ_VERSION: ${{ github.ref_name }}
+          DENO_DIR: "./deno"
+        run: |
+          BUILD=`git rev-parse --short "$GITHUB_SHA"`
+          TAQ_VERSION="${TAQ_VERSION/\//-}"
+          deno compile --output taq --allow-run --allow-write --allow-read --allow-env --allow-net --import-map ./import_map.json --no-prompt --target $DENO_TARGET index.ts --quickstart "`cat quickstart.md`" --setBuild "$BUILD" --setVersion "$TAQ_VERSION" --lock ./deno-lock.json
+          ./taq init -p ./test_project
```

Comment:
This is usually a test in bash after the compile step right? Is there a reason that this isn't like `[[ $(./taq init -p ./test_project) == "Project taq'ified!" ]] `?

Reply:
Thanks @alexzbusko, I reverted the change to remove that test